### DISCUSSION TITLE
Synchronizing 0/1 730 294

### DISCUSSION THREAD

**Discussion by roach20222** [Chia-Network/chia-blockchain#10839] [Support]
Status synchronization problem

Synchronizing 0/1 730 294. so remains, after update 1.3.1 what resolution.
![Capturar](https://user-images.githubusercontent.com/102169792/159514737-70cc3cde-2a9a-4fe6-b4bc-132b9dcde9e7.PNG)
thanks

**Comment by roach20222** [comment]
![Capturar1](https://user-images.githubusercontent.com/102169792/159527485-9f4a29cc-3315-4daa-8603-710bd2459eaf.PNG)

**Comment by Gooseboy2234** [comment]
Have you tried a DB and wallet file deletion? and reinstalled the chia client? What OS are you on? have you performed a DB upgrade?
